docs/troubleshooting.md

```diff
@@ -30,8 +30,9 @@ Secrets are masked out of the logs.
 
 ## Http Tracing Windows
 
-Start [Fiddler](http://www.telerik.com/fiddler).  
-It's recommended to only listen to agent traffic.  File, Capture Traffic off (F12) 
+Start [Fiddler](http://www.telerik.com/fiddler).<br />
```

Comment:
two spaces after the line is a CR in md.  

http://stackoverflow.com/questions/26188124/carriage-returns-in-markdown-section-of-ipython-notebook